Gaming The Syllabus/Assets/Scripts/Riddles/SyllabusRiddleManager.cs

```diff
@@ -10,6 +10,7 @@
 public class SyllabusRiddleManager : Singleton<SyllabusRiddleManager>
 {
     private static List<Riddle>? riddles;
+    private static HashSet<string>? solved_riddles;
```

Comment:
Good call with the HashSet! One thing: would it maybe be better to store the whole riddle instead of just the question? I'm just thinking of cases where maybe there are somehow riddles that share a question?

Reply:
Also, standard practice is to use camelCase in C#, so this should be named solvedRiddles to keep with that convention

Reply:
I don't know if the Riddle class is hashable, so let me see what I can do!

Reply:
You could make it Hashable I think! If it proves to be trouble lmk and I can work on it!